### PR TITLE
feat (api): application variable warnings

### PR DESCRIPTION
**Pull request by fsamin**
Linked to https://github.com/ovh/cds-ui/pull/30

Has been merged with https://github.com/ovh/cds/pull/199

### CODE REVIEW

File: engine/api/sanity/dao.go [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+	defer rows.Close()
+
+	var id int64
+	for rows.Next() {
+		var w sdk.Warning
+		var appID, pipID, envID, actionID, stageID sql.NullInt64
+		var appName, pipName, envName, actionName sql.NullString
+		var messageParam string
+
+		err := rows.Scan(&id, &w.ID, &messageParam,
```
Comment (sguiheux) [inline engine/api/sanity/dao.go]: error management in all file

File: engine/api/application.go [hunk 2]
```diff
@@ -576,6 +592,10 @@ func updateApplicationHandler(w http.ResponseWriter, r *http.Request, db *gorp.D
 	projectKey := vars["key"]
 	applicationName := vars["permApplicationName"]
 
+	p, err := project.LoadProject(db, projectKey, c.User)
```
Comment (sguiheux) [inline engine/api/application.go]: check error ?

File: engine/api/application.go [hunk 3]
```diff
@@ -576,6 +592,10 @@ func updateApplicationHandler(w http.ResponseWriter, r *http.Request, db *gorp.D
...
+	envs, err := environment.LoadEnvironments(db, projectKey, true, c.User)
```
Comment (sguiheux) [inline engine/api/application.go]: idem

File: engine/api/application.go (hunk 2 above)
Comment (fsamin) [inline engine/api/application.go]: fixed https://github.com/ovh/cds/pull/195/commits/9db3bdc6fedf4a906c61f09c569e66c255da117b

File: engine/api/application.go (hunk 3 above)
Comment (fsamin) [inline engine/api/application.go]: fixed https://github.com/ovh/cds/pull/195/commits/9db3bdc6fedf4a906c61f09c569e66c255da117b

File: engine/api/sanity/dao.go (hunk 1 above)
Comment (fsamin) [inline engine/api/sanity/dao.go]: fixed https://github.com/ovh/cds/pull/195/commits/9db3bdc6fedf4a906c61f09c569e66c255da117b

**Pull request merged**